15_1_baloon.js

```diff
@@ -0,0 +1,55 @@
+const baloon = document.querySelector('.baloon');
+const measure = document.querySelector('.measure');
+const btnStart = document.querySelector('.start');
+
+let size = 50;
+let level = -200;
+
+const playGame = () => {
+    clearScore();
+    window.addEventListener('keydown', pump);
+};
+
+const pump = event => {
+    if (event.keyCode === 38) {
+        maximize();
+    }
+
+    if (event.keyCode === 40) {
+        minimize();
+    }
+
+    if (baloon.style.fontSize === '250px') {
```

Comment:
why 250 ?

Reply:
there is no particular reason. I've chosen the maximus size.